io.lmms.LMMS.appdata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 the authors -->
```

Comment:
> the authors

I am really not sure about this...

Reply:
> > the authors

> I am really not sure about this...

[`"LMMS Developers"`](https://github.com/LMMS/lmms/blob/stable-1.2/CMakeLists.txt#L25) is what we use elsewhere.  Upstream, this would be a good candidate as a [`cmake` configured file](https://github.com/LMMS/lmms/blob/stable-1.2/cmake/linux/CMakeLists.txt#L8), e.g.

```xml
<!-- ${PROJECT_COPYRIGHT} -->
...
 <url type="homepage">${PROJECT_URL}</url>
```

... etc.

Reply:
I'll work on a patch for upstream.